ToDoList/ToDoProgram/ToDoList.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ToDoListProgram
+{
+    public class ToDoList
+    {
+        public int idCounter = 1;
+
+        public List<ToDo> list;
+
+        public ToDoList()
+        {
+            list = new List<ToDo>();
```

Comment:
Do we need to initialize the list in the constructor if we are not passing its value